api/app/auto_spatial_advisory/elevation.py

```diff
@@ -178,6 +211,75 @@ def apply_threshold_mask_to_dem(threshold: int, mask_path: str, temp_dir: str):
     return masked_dem_path
 
 
+@dataclass(frozen=True)
+class FireZoneTPIStats:
+    """
+    Captures fire zone stats of TPI pixels hitting >4K HFI threshold via
+    a dictionary, fire_zone_stats, of {source_identifier: {1: X, 2: Y, 3: Z}}, where 1 = valley bottom, 2 = mid slope, 3 = upper slope
```

Comment:
It's kinda awkward because we rely on `np.unique` for classified counts. So we'd have to add an extra bunch of code to make the integers enums. Not sure if it's worth it.

Reply:
Ok, I'm fine with leaving it as is.